#872 update volumes_15min_mvt pkey

gabrielwol opened the pull request:
## What this pull request accomplishes:
- Removes `volume_15min_mvt_uid` from pkey of `volumes_15min_mvt` so that it works as a unique constraint.

## Issue(s) this solves:
- #872 

## What, in particular, needs to reviewed:

- 

## What needs to be done by a sysadmin after this PR is merged
- [X] duplicates deleted by clearing + re-aggregating
- [ ] Alter pkey:
```sql
ALTER TABLE miovision_api.volumes_15min_mvt DROP CONSTRAINT volumes_15min_mvt_int_uid_dt_bin_class_leg_mvmt_uid_pkey PRIMARY KEY (volume_15min_mvt_uid, intersection_uid, datetime_bin, classification_uid, leg, movement_uid);

ALTER TABLE miovision_api.volumes_15min_mvt ADD CONSTRAINT volumes_15min_mvt_int_dt_bin_class_leg_mvmt_uid_pkey PRIMARY KEY (intersection_uid, datetime_bin, classification_uid, leg, movement_uid);
```
Surprisingly, I tried this in my personal schema and it works, despite temporarily dropping the pkey used for partitioning.

chmnata commented:
ran 
```
ALTER TABLE IF EXISTS miovision_api.volumes_15min_mvt DROP CONSTRAINT IF EXISTS volumes_15min_mvt_int_uid_dt_bin_class_leg_mvmt_uid_pkey;
```
 instead of 
```
ALTER TABLE miovision_api.volumes_15min_mvt 
DROP CONSTRAINT volumes_15min_mvt_int_uid_dt_bin_class_leg_mvmt_uid_pkey 
PRIMARY KEY(volume_15min_mvt_uid, intersection_uid, datetime_bin, classification_uid, leg, movement_uid);
```

gabrielwol commented:
Fixed the remaining duplicates by reaggregating `python3 intersection_tmc.py run-api --intersection 66 --start_date 2023-10-03 --end_date 2024-04-10 --agg --path /data/airflow/data_scripts/volumes/miovision/api/config.cfg` and implemented the Pkey change ✅
